furusiya/function_mess.py

```diff
@@ -140,24 +122,20 @@ def process_bow():
                 elif event.type == 'KEYDOWN':
                     if event.key == 'ESCAPE':
                         Game.draw_bowsight = False
-                        is_cancelled = True
+                        break
                     elif event.char == 'f':
                         if Game.target and Game.target.has_component(Fighter):
                             is_critical = False
                             damage_multiplier = config.data.weapons.arrowDamageMultiplier
-                            if config.data.features.bowCrits and randint(0,
-                                                                         100) <= config.data.weapons.bowCriticalProbability:
-                                damage_multiplier *= (1 + config.data.weapons.bowCriticalDamageMultiplier)
+                            if config.data.features.bowCrits and randint(0, 100) <= config.data.weapons.bowCriticalProbability:
+                                damage_multiplier *= 1 + config.data.weapons.bowCriticalDamageMultiplier
```

Comment:
This must be the most mysterious and inexplicable code I've written for this game so far.